site/docs/v1/tech/events-webhooks/event-bodies/user-registration-delete-complete.adoc

```diff
@@ -0,0 +1,52 @@
+:type: user.registration.delete.complete
+== User Registration Delete Complete
+
+[NOTE.since]
+====
+This event has been available since 1.30.0
+====
+
+This event is generated when a user registration delete transaction has completed.
+
+[source,shell]
+.Event type
+[subs="attributes"]
+----
+{type}
+----
+
+=== Event Scope
+
+include::../_tenant-or-application-scoped-event.adoc[]
+
```

Comment:
Add `Transaction Compatible` section. This is, I presume, not transaction compatible.